src/components/sidebar/index.js

```diff
@@ -5,19 +5,19 @@
 
```

Comment:
As usual, the changes in this file won't be merged in this PR. After approval I'll remove the commit with this change and if necessary regenerate the various test snapshots that are affected.

I enabled the sidebar in 2 different panels so that we can test that the state is per panel.